kafka-junit-core/src/main/java/com/salesforce/kafka/test/KafkaTestServer.java

```diff
@@ -76,19 +77,44 @@
      */
     private final String localHostname;
 
+    /**
+     * Defines overridden broker properties.
+     */
+    private final Properties overrideBrokerProperties = new Properties();
```

Comment:
Why not just call this brokerProperties?

Reply:
I thought it was more descriptive, since you do not have provide a complete configuration, just whatever properties you want to set with a fall back to the libraries defaults.

Thoughts?